src/main/kotlin/com/github/chriskn/structurizrextension/internal/icons/IconRegistry.kt

```diff
@@ -82,7 +82,8 @@ object IconRegistry {
         "postgresql" to "${GILBARBARA_ICON_URL}postgresql.puml",
         "react" to "${GILBARBARA_ICON_URL}react.puml",
         "redis" to "${GILBARBARA_ICON_URL}redis.puml",
-        "kafka" to "${GILBARBARA_ICON_URL}kafka.puml",
+        "kafka" to "${GILBARBARA_ICON_URL}rabbitmq.puml",
```

Comment:
The icon paths for kafka and rabbitmq are swapped.

Reply:
apologies for the embarrassing mistake - BTW : thanks for the efforts with the extensions ;-)

Reply:
No problem, you are welcome 👍